### ISSUE TITLE
About the first stage and second stage

### ISSUE THREAD

**Issue by clearlyzerolxd** [mueller-franzes/medfusion#23]
Hello, I have some questions about the way to normalize data. Should normalization be done this way to get good results or is it like this?

```python
T.ToTensor(),

T.Normalize(mean=0.5, std=0.5)
```

or 
```python 
T.ToTensor()
data = (data -1)/2
```

**Comment by clearlyzerolxd** [comment]
![截图 2023-12-10 13-44-52](https://github.com/mueller-franzes/medfusion/assets/128237886/c7372405-7595-436f-852b-510086cc72e4)

**Comment by clearlyzerolxd** [comment]
![截图 2023-12-10 13-46-42](https://github.com/mueller-franzes/medfusion/assets/128237886/50b1ff8b-e8ec-4423-a474-65e1fd2305ca)
The reconstruction effect is still a bit blurry. Is this trend correct?

**Comment by clearlyzerolxd** [comment]
I rewrote the second stage code using pytorch according to the author's ideas. I found that it didn't seem to converge well.

**Comment by clearlyzerolxd** [comment]
The idea is roughly like this

```
            with torch.no_grad():
                out = vae.module.encode(data.to(device))
            loss = diffusion(out,None)
            net_optim.zero_grad()
            loss.backward()
            net_optim.step()
```
